packages/notionable/src/global/account/communities.ts

```diff
@@ -5,46 +5,67 @@ import {
 } from '@notional-finance/util';
 import { Contract } from 'ethers';
 
-export type CommunityName = 'DEGEN_SCORE' | 'V3_BETA_CONTEST';
+export type CommunityName = 'DEGEN_SCORE' | 'V3_BETA_CONTEST' | 'L2DAO' | 'CRYPTO_TESTERS' | 'LLAMAS';
```

Comment:
Can we use these names in the other places where we have community strings?

Reply:
Not sure what you're wanting to do here. Do you want to make this a enum so we can use them as keys in the CommunityNFTs object and else where? I think it would be a pain to have a function that parses those strings to the display strings.

Reply:
Yeah maybe just make it an enum then